app/views/content/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach.md

```diff
@@ -3,8 +3,9 @@ title: Age groups you could teach
 heading: Age groups you could teach
 description: |-
   Explore the key stages and ages you could teach, whether that's primary, secondary, early years, college, or pupils with special educational needs.
-promo_content:
-    - content/train-to-be-a-teacher/promos/mailing-list-promo-who-do-you-want-to-teach
+cta_mailinglist:
+  mailinglist:
+    text: Explore how you can get into teaching primary or secondary, including top tips on making a successful application.
```

Comment:
I know this is in prod already but I find `Explore` a bit odd here - you get sent emails, rather than exploring yourself?

Is there a better way to word this?